doc/contribute/bin_blobs.rst

```diff
@@ -0,0 +1,201 @@
+.. _bin-blobs:
+
+.. warning::
+
+   This page documents Zephyr's support for binary blobs. The tooling and
+   infrastructure for this purpose are still incomplete, and so this page may
+   refer to future additions.
+
+Binary Blobs
+************
+
+In the context of an operating system that supports multiple architectures and
+many different IC families, some functionality may be unavailable without the
+help of executable code distributed in binary form.
+Binary blobs (or blobs for short) are files containing machine code or data in
+proprietary formats, e.g. without corresponding source code released under an
+OSI approved license.
+
+Zephyr supports the distribution and use of binary blobs via its built-in
+mechanisms, with some important caveats, described in the following sections. It
+is important to note that all the information in this section applies only to
+upstream or vanilla Zephyr. There are no limitations whatsoever (except perhaps
+license compatibility) in the support for binary blobs in forks or third-party
+distributions of Zephyr. In fact, Zephyr’s build system supports arbitrary use
+cases related to blobs. This includes linking against libraries, flashing images
+to targets, etc. Users are therefore free to create Zephyr-based downstream
+software which uses binary blobs if they cannot meet the requirements described
+in this page.
+
+Software license
+================
+
+Binary blobs do not typically use OSI approved licenses such as the Apache-2.0
+license. Instead most blobs are distributed under proprietary licenses which
+vary wildly in nature and conditions.
+Licenses will be approved individually by the Technical Steering Committe (TSC)
+upon request of inclusion of a binary blob.
+Blob vendors may impose a click-through or other EULA-like workflow when users
```

Comment:
Interesting question but it's not clear whether automation and validation are supposed part of this document or not. There is something about a test suite in the "Limited Scope" below but it looks like "work in progress". Maybe it would be more practical to discuss separately?